src/platform/plugins/shared/navigation/public/top_nav_menu/top_nav_menu.tsx

```diff
@@ -100,18 +106,26 @@ export function TopNavMenu<QT extends AggregateQuery | Query = Query>(
 
   function renderLayout() {
     const { setMenuMountPoint, visible } = props;
-    const menuClassName = classNames('kbnTopNavMenu', props.className);
-    const wrapperClassName = classNames('kbnTopNavMenu__wrapper', {
-      'kbnTopNavMenu__wrapper--hidden': visible === false,
-    });
+    const styles = {
+      withElements: css`
+        display: flex;
+        align-items: center;
+      `,
```

Comment:
nit: could we give this a different name, it used to be called `kbnTopNavMenu__badgeWrapper`, so it might easier to debug this in the future?

Reply:
Sure, so just something like this?
```suggestion
      badgeWrapper: css`
        display: flex;
        align-items: center;
      `,
```
?